datasets/export/download_export_archive.py

```diff
@@ -0,0 +1,17 @@
+"""
+This is an example of how to download the contents in a dataset version export archive. For example, if the dataset export contains images, this script will download
+the images from the hosted URLs and produce a directory that contains all image files by the dataset splits.
+
+Before running this script, please make sure the following has been completed:
+1. A dataset version export (currently only supports protobuf) has been created
+2. An API KEY for the app has been generated.
```

Comment:
A minor change. It could be replaced with
2. `CLARIFAI_PAT` should be set as Env Variable

Reply:
Done